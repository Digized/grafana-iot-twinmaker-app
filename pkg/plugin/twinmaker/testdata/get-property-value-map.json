{
    "PropertyValues": {
        "documents": {
            "PropertyReference": {
                "ComponentName": "SpecSheets",
                "EntityId": "Mixer_1_4b57cbee-c391-4de6-b882-622c633a697e",
                "ExternalIdProperty": null,
                "PropertyName": "documents"
            },
            "PropertyValue": {
                "BooleanValue": null,
                "DoubleValue": null,
                "Expression": null,
                "IntegerValue": null,
                "ListValue": null,
                "LongValue": null,
                "MapValue": {
                    "manufacturer_page": {
                        "BooleanValue": null,
                        "DoubleValue": null,
                        "Expression": null,
                        "IntegerValue": null,
                        "ListValue": null,
                        "LongValue": null,
                        "MapValue": null,
                        "RelationshipValue": null,
                        "StringValue": "https://console.aws.amazon.com/iottwinmaker/home"
                    },
                    "mixer1_manual": {
                        "BooleanValue": null,
                        "DoubleValue": null,
                        "Expression": null,
                        "IntegerValue": null,
                        "ListValue": null,
                        "LongValue": null,
                        "MapValue": null,
                        "RelationshipValue": null,
                        "StringValue": "https://github.com/aws-samples/aws-iot-twinmaker-samples"
                    }
                },
                "RelationshipValue": null,
                "StringValue": null
            }
        }
    }
}