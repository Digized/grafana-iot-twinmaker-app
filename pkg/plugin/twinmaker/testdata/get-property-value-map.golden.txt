ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ

Frame[0] 
Name: 
Dimensions: 2 Fields by 2 Rows
+-------------------+----------------------------------------------------------+
| Name: Key         | Name: Value                                              |
| Labels:           | Labels:                                                  |
| Type: []*string   | Type: []*string                                          |
+-------------------+----------------------------------------------------------+
| manufacturer_page | https://console.aws.amazon.com/iottwinmaker/home         |
| mixer1_manual     | https://github.com/aws-samples/aws-iot-twinmaker-samples |
+-------------------+----------------------------------------------------------+


====== TEST DATA RESPONSE (arrow base64) ======
FRAME=QVJST1cxAAD/////uAEAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEEAAoADAAAAAgABAAKAAAACAAAAFAAAAACAAAAKAAAAAQAAADE/v//CAAAAAwAAAAAAAAAAAAAAAUAAAByZWZJZAAAAOT+//8IAAAADAAAAAAAAAAAAAAABAAAAG5hbWUAAAAAAgAAANgAAAAEAAAAQv///xQAAACkAAAApAAAAAAABQGgAAAAAgAAACwAAAAEAAAANP///wgAAAAQAAAABQAAAFZhbHVlAAAABAAAAG5hbWUAAAAAWP///wgAAABQAAAARwAAAHsibGlua3MiOlt7InRpdGxlIjoiTGluayIsInRhcmdldEJsYW5rIjp0cnVlLCJ1cmwiOiIke19fdmFsdWUudGV4dH0ifV19AAYAAABjb25maWcAAAAAAACU////BQAAAFZhbHVlABIAGAAUABMAEgAMAAAACAAEABIAAAAUAAAAQAAAAEQAAAAAAAUBQAAAAAEAAAAMAAAACAAMAAgABAAIAAAACAAAAAwAAAADAAAAS2V5AAQAAABuYW1lAAAAAAAAAAAEAAQABAAAAAMAAABLZXkA/////9gAAAAUAAAAAAAAAAwAFgAUABMADAAEAAwAAACoAAAAAAAAABQAAAAAAAADBAAKABgADAAIAAQACgAAABQAAAB4AAAAAgAAAAAAAAAAAAAABgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAwAAAAAAAAAEAAAAAAAAAAeAAAAAAAAADAAAAAAAAAAAAAAAAAAAAAwAAAAAAAAAAwAAAAAAAAAQAAAAAAAAABoAAAAAAAAAAAAAAACAAAAAgAAAAAAAAAAAAAAAAAAAAIAAAAAAAAAAAAAAAAAAAAAAAAAEQAAAB4AAAAAAAAAbWFudWZhY3R1cmVyX3BhZ2VtaXhlcjFfbWFudWFsAAAAAAAAMAAAAGgAAAAAAAAAaHR0cHM6Ly9jb25zb2xlLmF3cy5hbWF6b24uY29tL2lvdHR3aW5tYWtlci9ob21laHR0cHM6Ly9naXRodWIuY29tL2F3cy1zYW1wbGVzL2F3cy1pb3QtdHdpbm1ha2VyLXNhbXBsZXMQAAAADAAUABIADAAIAAQADAAAABAAAAAsAAAAPAAAAAAABAABAAAAyAEAAAAAAADgAAAAAAAAAKgAAAAAAAAAAAAAAAAAAAAAAAAAAAAKAAwAAAAIAAQACgAAAAgAAABQAAAAAgAAACgAAAAEAAAAxP7//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAADk/v//CAAAAAwAAAAAAAAAAAAAAAQAAABuYW1lAAAAAAIAAADYAAAABAAAAEL///8UAAAApAAAAKQAAAAAAAUBoAAAAAIAAAAsAAAABAAAADT///8IAAAAEAAAAAUAAABWYWx1ZQAAAAQAAABuYW1lAAAAAFj///8IAAAAUAAAAEcAAAB7ImxpbmtzIjpbeyJ0aXRsZSI6IkxpbmsiLCJ0YXJnZXRCbGFuayI6dHJ1ZSwidXJsIjoiJHtfX3ZhbHVlLnRleHR9In1dfQAGAAAAY29uZmlnAAAAAAAAlP///wUAAABWYWx1ZQASABgAFAATABIADAAAAAgABAASAAAAFAAAAEAAAABEAAAAAAAFAUAAAAABAAAADAAAAAgADAAIAAQACAAAAAgAAAAMAAAAAwAAAEtleQAEAAAAbmFtZQAAAAAAAAAABAAEAAQAAAADAAAAS2V5AOgBAABBUlJPVzE=
